### PR TITLE
Replacing Rudr with Crossplane

### PR DESCRIPTION
**Pull request by vturecek**
Replacing Rudr with Crossplane as the Kubernetes implementation. Also updating the sample configs to the alpha 2 spec.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -122,7 +122,7 @@ <h1 class="title">A team-centric <strong>standard</strong> for building cloud na
       <div class="column is-full">
           <div class="button-cta has-text-centered">
             <a class="button is-primary is-large" href="https://github.com/oam-dev/spec">Read the Spec</a>
-            <p class="rudr-cta">Check out <a href="https://github.com/oam-dev/rudr"><strong>Rudr</strong></a> for a Kubernetes implementation of the specification. &darr;</p>
+            <p class="rudr-cta">Check out <a href="https://github.com/crossplane/crossplane"><strong>Crossplane</strong></a> for a Kubernetes implementation of the specification. &darr;</p>
```
Comment (prasek) [inline public/index.html:125]:
```suggestion
            <p class="rudr-cta">Check out <a href="https://github.com/crossplane/crossplane"><strong>Crossplane</strong></a> for the Kubernetes implementation of the specification. &darr;</p>

```

File: public/index.html [hunk 2]
```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
...
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/rudr.svg" alt="Rudr" /></div>
+      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
       <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
-        <small><a href="https://github.com/oam-dev/rudr">github.com/oam-dev/rudr</a></small>
+        <h1>Crossplane: A Kubernetes implementation of the Open Application Model</h1>
```
Comment (prasek) [inline public/index.html:289]:
```suggestion
        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
```

File: themes/clean-landing/layouts/partials/hero.html [hunk 3]
```diff
@@ -40,7 +40,7 @@ <h1 class="title">A team-centric <strong>standard</strong> for building cloud na
       <div class="column is-full">
           <div class="button-cta has-text-centered">
             <a class="button is-primary is-large" href="https://github.com/oam-dev/spec">Read the Spec</a>
-            <p class="rudr-cta">Check out <a href="https://github.com/oam-dev/rudr"><strong>Rudr</strong></a> for a Kubernetes implementation of the specification. &darr;</p>
+            <p class="rudr-cta">Check out <a href="https://github.com/crossplane/crossplane"><strong>Crossplane</strong></a> for a Kubernetes implementation of the specification. &darr;</p>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/hero.html:43]:
```suggestion
            <p class="rudr-cta">Check out <a href="https://github.com/crossplane/crossplane"><strong>Crossplane</strong></a> for the Kubernetes implementation of the specification. &darr;</p>

```

File: themes/clean-landing/layouts/partials/implementations.html [hunk 4]
```diff
@@ -3,50 +3,48 @@
...
     </div>
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/rudr.svg" alt="Rudr" /></div>
+      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
       <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
-        <small><a href="https://github.com/oam-dev/rudr">github.com/oam-dev/rudr</a></small>
+        <h1>Crossplane: A Kubernetes implementation of the Open Application Model</h1>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/implementations.html:13]:
```suggestion
        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
```

File: themes/clean-landing/layouts/partials/implementations.html [hunk 5]
```diff
@@ -3,50 +3,48 @@
...
     </div>
 
     <div class="columns">
       <div class="column is-quarter has-text-left">
         <ul>
-          <li><a href="https://github.com/oam-dev/rudr/tree/master/docs">Documentation</a></li>
-          <li><a href="https://github.com/oam-dev/rudr/blob/master/docs/setup/install.md">Getting Started</a></li>
-          <li><a href="https://github.com/oam-dev/rudr">Rudr Github Repo</a></li>
+          <li><a href="https://crossplane.io/docs/latest">Documentation</a></li>
+          <li><a href="https://github.com/crossplane/crossplane/blob/master/docs/getting-started/install.md">Getting Started</a></li>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/implementations.html:23]:
```suggestion
          <li><a href="https://crossplane.io/docs/latest">Getting Started</a></li>
```
The docs are changing so best to link to the landing page.

File: public/index.html [hunk 6]
```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
...
     </div>
 
     <div class="columns">
       <div class="column is-quarter has-text-left">
         <ul>
-          <li><a href="https://github.com/oam-dev/rudr/tree/master/docs">Documentation</a></li>
-          <li><a href="https://github.com/oam-dev/rudr/blob/master/docs/setup/install.md">Getting Started</a></li>
-          <li><a href="https://github.com/oam-dev/rudr">Rudr Github Repo</a></li>
+          <li><a href="https://crossplane.io/docs/latest">Documentation</a></li>
+          <li><a href="https://github.com/crossplane/crossplane/blob/master/docs/getting-started/install.md">Getting Started</a></li>
```
Comment (prasek) [inline public/index.html:299]:
```suggestion
          <li><a href="https://crossplane.io/docs/latest">Getting Started</a></li>
```
Docs are changing so best to link to the docs landing page.

File: public/index.html [hunk 7]
```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
...
+          <li><a href="https://github.com/crossplane">Crossplane on GitHub</a></li>
```
Comment (prasek) [inline public/index.html:300]:
```suggestion
          <li><a href="https://github.com/crossplane/crossplane">Crossplane on GitHub</a></li>
```

File: public/index.html [hunk 8]
```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
...
         </ul>
       </div>
       <div class="column is-three-quarters">
 
-        <p><a href="https://github.com/oam-dev/rudr">Rudr</a> takes an incremental approach to solving problems. The architecture is a set of plugins for Kubernetes which allow OAM specifications to be implemented and deployed on clusters with native APIs <em>(you still use kubectl)</em>.</p>
-
-        <p>This allows app developers to focus on building OAM components, app operators to focus on operational capabilities through the OAM app config and infra operators to focus on Kubernetes. By leveraging the Open App Model, users now have a framework to define their apps on their Kubernetes clusters.</p>
+        <p><a href="https://github.com/crossplane">Crossplane</a> is an open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.</p>
```
Comment (prasek) [inline public/index.html:305]:
```suggestion
        <p><a href="https://github.com/crossplane/crossplane">Crossplane</a> is an open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.</p>
```

File: public/index.html [hunk 9]
```diff
@@ -287,50 +279,48 @@ <h4>Show a component and a config in one file:</h4>
...
-        <p>Currently, Rudr will leverage the defined trait to accomplish the task. This gives the freedom to use whatever underlying tool the user wants while providing a trait that focuses on the functionality and not the technology. In the future, Rudr might provide a set of default technologies to provide the functionality desired by a trait.</p>
+        <p>This allows application developers to focus on writing code, app operators to focus on operational capabilities through the OAM app config, and infra operators to build out curated infrastructure and custom APIs using familiar Kubernetes tools and APIs.</p>
       </div>
     </div>
     <div class="columns">
       <div class="column is-quarter has-text-centered">
         <hr>
         
-        <p><a href="https://github.com/oam-dev/rudr" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
+        <p><a href="https://github.com/crossplane" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
```
Comment (prasek) [inline public/index.html:314]:
```suggestion
        <p><a href="https://github.com/crossplane/crossplane" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
```

File: themes/clean-landing/layouts/partials/implementations.html [hunk 10]
```diff
@@ -3,50 +3,48 @@
...
+          <li><a href="https://github.com/crossplane">Crossplane on GitHub</a></li>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/implementations.html:24]:
```suggestion
          <li><a href="https://github.com/crossplane/crossplane">Crossplane on GitHub</a></li>
```

File: themes/clean-landing/layouts/partials/implementations.html [hunk 11]
```diff
@@ -3,50 +3,48 @@
...
         </ul>
       </div>
       <div class="column is-three-quarters">
 
-        <p><a href="https://github.com/oam-dev/rudr">Rudr</a> takes an incremental approach to solving problems. The architecture is a set of plugins for Kubernetes which allow OAM specifications to be implemented and deployed on clusters with native APIs <em>(you still use kubectl)</em>.</p>
+        <p><a href="https://github.com/crossplane">Crossplane</a> is an open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.</p>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/implementations.html:29]:
```suggestion
        <p><a href="https://github.com/crossplane/crossplane">Crossplane</a> is an open source Kubernetes add-on that enables you to provision and manage OAM applications and the infrastructure they run on directly from Kubernetes.</p>
```

File: themes/clean-landing/layouts/partials/implementations.html [hunk 12]
```diff
@@ -3,50 +3,48 @@
...
-        <p>Currently, Rudr will leverage the defined trait to accomplish the task. This gives the freedom to use whatever underlying tool the user wants while providing a trait that focuses on the functionality and not the technology. In the future, Rudr might provide a set of default technologies to provide the functionality desired by a trait.</p>
+        <p>This allows application developers to focus on writing code, app operators to focus on operational capabilities through the OAM app config, and infra operators to build out curated infrastructure and custom APIs using familiar Kubernetes tools and APIs.</p>
       </div>
     </div>
     <div class="columns">
       <div class="column is-quarter has-text-centered">
         <hr>
         
-        <p><a href="https://github.com/oam-dev/rudr" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
+        <p><a href="https://github.com/crossplane" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
```
Comment (prasek) [inline themes/clean-landing/layouts/partials/implementations.html:38]:
```suggestion
        <p><a href="https://github.com/crossplane/crossplane" class="button is-primary is-large"><span class="icon"><i class="fa fa-github"></i></span> &nbsp; Learn More</a></p>
```

**Comment by vturecek** [comment]
Thanks @prasek! Added your changes.

**Comment by jbw976** [comment]
@vturecek do you have an ETA for when this PR can get merged? I'm excited to see these changes on the website :)

File: themes/clean-landing/layouts/partials/implementations.html [hunk 13]
```diff
@@ -78,11 +76,12 @@ <h1>Upcoming Implementations</h1>
       </div>
       <div class="column is-three-quarters">
 
-        <p>A variety of parterns working on adding support for Open Application Model in their offerings:</p>
+        <p>There are a variety of projects working on adding support for Open Application Model in their offerings.</p>
```
[the remark this answers is not in the document]
Comment (resouer) [reply themes/clean-landing/layouts/partials/implementations.html:79]: This one can be fixed in follow up PR if it's rush.

File: themes/clean-landing/layouts/partials/implementations.html [hunk 14]
```diff
@@ -3,50 +3,48 @@
...
 
     <div class="columns">
-      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/rudr.svg" alt="Rudr" /></div>
+      <div class="column is-quarter is-one-quarter-mobile has-text-left"><img src="images/crossplane.png" alt="Crossplane" /></div>
       
       <div class="column is-three-quarters is-three-quarters-mobile">
-        <h1>Rudr: A Kubernetes Implementation of the Open Application Model</h1>
-        <small><a href="https://github.com/oam-dev/rudr">github.com/oam-dev/rudr</a></small>
+        <h1>Crossplane: The Kubernetes implementation of the Open Application Model</h1>
+        <small><a href="https://crossplane.io">crossplane.io</a></small>
```
[the remark this answers is not in the document]
Comment (vturecek) [reply themes/clean-landing/layouts/partials/implementations.html:14]: There is a Crossplane on GitHub link on the left for that, I think this one should go to the landing page.

File: themes/clean-landing/layouts/partials/implementations.html (hunk 13 above)
[the remark this answers is not in the document]
Comment (vturecek) [reply themes/clean-landing/layouts/partials/implementations.html:79]: OK, let me open up a second PR for that.

File: themes/clean-landing/layouts/partials/implementations.html (hunk 14 above)
[the remark this answers is not in the document]
Comment (prasek) [reply themes/clean-landing/layouts/partials/implementations.html:14]: Yes, crossplane.io is already pointing to https://oam.dev so we should be good to go, thanks!

**Pull request merged**